radix-engine-toolkit/src/visitor/value/aliasing_visitor.rs

```diff
@@ -85,13 +88,19 @@ impl ManifestAstValueVisitor for ValueAliasingVisitor {
             ManifestAstValue::Address { address } => {
                 match address {
                     EntityAddress::ComponentAddress { address } => {
-                        *value = ManifestAstValue::ComponentAddress { address: *address };
+                        *value = ManifestAstValue::Address {
```

Comment:
The `visit_address` bit can be safely removed as it currently does nothing. Previously, it was used to convert `Address` to `[Package|Resource|Component]Address` types.